x-pack/filebeat/docs/inputs/input-entity-analytics.asciidoc

```diff
@@ -0,0 +1,116 @@
+[role="xpack"]
+
+:type: entity-analytics
+
+[id="{beatname_lc}-input-{type}"]
+=== Entity Analytics Input
+
+++++
+<titleabbrev>Entity Analytics</titleabbrev>
+++++
+
+Experimental[Input]
+
+The Entity Analytics input collects identity assets, such as users, from external
+identity providers.
+
+The following identity providers are supported:
+
+- <<provider-azure>>
+
+==== Configuration options
+
+The `entity-analytics` input supports the following configuration options plus the
+<<{beatname_lc}-input-{type}-common-options>> described later.
+
+[float]
+==== `provider`
+
+The identity provider. Must be one of: `azure`.
```

Comment:
Should the provider name be more specific, like `azure_ad`? Let's say in the future we want to collect host entities from Azure with this? Would that be added into the existing provider or would be create a new provider like azure_vm or azure_aks to collect those entities?

Reply:
Great question... I suppose that depends on if we want to keep users/groups separate from hosts. If that isn't a concern, and we can use the same API, then putting it under the same provider may not be a bad idea.

Either way, I'll switch it to `azure_ad` since it's a bit more specific, and saves us the annoyance of trying to change it later if we do add `azure_vm`, `azure_aks`, or anything else.